e2e/cypress/support/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare namespace Cypress {
```

Comment:
Do we need this file twice? 😃

Reply:
unfortunately we weren't able to figure out a way to use `cy.login` inside the `adminLogin` command definition without duplicating the `index.d.ts` under in the `support` dir.

we could also do away with the `admin` login helper

Reply:
the main issue is that adding cypress custom commands in typescript feels very brittle at this point. there's dozens of different ways to do it according to [this issue](https://github.com/cypress-io/cypress/issues/1065), and even [the example](https://github.com/omerose/cypress-support/blob/master/cypress/support/commands.ts) the cypress docs point to didn't "just work" as expected. only adding the `index.d.ts` finally got the spec files to recognize the custom commands.

this approach might be more stable in the longrun: https://medium.com/@NicholasBoll/cypress-io-scaling-e2e-testing-with-custom-commands-6b72b902aab

Reply:
actually looks like [this comment](https://github.com/cypress-io/cypress/issues/1152#issuecomment-509729392) solved the problem!